sdk/mediaservices/Azure.Media.Analytics.Edge/src/Customization/MediaGraphInstance.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.Media.Analytics.Edge.Models
+{
+    public partial class MediaGraphInstance
+    {
+        /// <summary>
+        ///  Deserialize MediaGraphInstance.
+        /// </summary>
+        /// <param name="json"></param>
+        /// <returns></returns>
```

Comment:
nit: docs missing for `returns`. 

Can we add docs for public methods/properties in other places as well, so that it can generate properly in the documentation?